From 62944b09b91be971008b91f2e40b0120aac9695e Mon Sep 17 00:00:00 2001
From: MVZ Ludwigsburg <edv@mvz-labor-lb.de>
Date: Wed, 14 Dec 2022 16:57:24 +0100
Subject: [PATCH] hwdb: Add mount matrix for CSL Panther Tab HD

(cherry picked from commit a76d7acae1cfb9951041b31b3c9db2d8c1ebd8a7)

Related: RHEL-5950
---
 hwdb.d/60-sensor.hwdb | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index 89119247fa..07b1c1625e 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -283,6 +283,13 @@ sensor:modalias:acpi:BOSC0200*:dmi:*:svnHampoo*:pnC3W6_AP108_4GB:*
 sensor:modalias:acpi:KIOX000A*:dmi:*:svnConnect:pnTablet9:*
  ACCEL_MOUNT_MATRIX=0, 1, 0; 1, 0, 0; 0, 0, 1
 
+#########################################
+# CSL Computer
+#########################################
+# CSL Panther Tab HD
+sensor:modalias:acpi:KIOX000A*:dmi:*:svnCSL*Computer*:pnCSL*Panther*Tab*HD:*
+ ACCEL_MOUNT_MATRIX=1, 0, 0; 0, -1, 0; 0, 0, 1
+
 #########################################
 # Cube
 #########################################
